src/internal/operations/backup.go

```diff
@@ -739,6 +765,10 @@ func (op *BackupOperation) persistResults(
 
 	op.Results.ItemsRead = opStats.ctrl.Successes
 
+	if op.Status == Completed {
```

Comment:
Is it possible for a backup to be completed and also have a non-nil op.Errors.Failure()?

Reply:
also a question: do we only want success/failure states? Are backups that succeeded with errors considered successful? How will we display that to the users?

Reply:
> Is it possible for a backup to be completed and also have a non-nil op.Errors.Failure()?
- I have changed this logic a bit. Could you please re-review? 

> Are backups that succeeded with errors considered successful? How will we display that to the users?
- This needs some discussion. I'll prepare some material to discuss on Monday.